Program.cs

```diff
@@ -470,66 +472,123 @@ static P2pEndPoint GetExternalEndPoint(Socket socket)
                     Internal = (socket.LocalEndPoint as IPEndPoint)
                 };
             }
-
+            
             Console.WriteLine("Could not find a working STUN server");
+
             return null;
         }
 
-
         static int SleepTime(DateTime now)
         {
-            List<int> seconds = new List<int>() {10, 20, 30, 40, 50, 60};

```

Comment:
any reason why moving this line of code? seems it's not used anywhere else so it doesn't need to move

Reply:
I guess I failed the test.

Reply:
hehe TBH I don't know yet if you passed the test because I didn't run your code yet (but I will soon); I'm just being very picky with the changes to get the PR ready to be proposed to the original repo

Reply:
I guess there will not be any difference between the result of code reviews and running the application, don't expect much from hitting F5.

Reply:
I didn't say that I expect a different behaviour from before the reviews to after the reviews, I mean that I even haven't run your code yet